CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.3.0
+
+A breaking release that updates scale-info from v2.7 to 2.11 because it's exposed in the public API, this needed to sync up with other scale-info based projects.
```

Comment:
Let's see if we need this first because 2.7.x should be compatible up to <3.0.0 anyways :)

Reply:
(https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html)